src/main/java/de/unistuttgart/informatik/fius/icge/workbench/swing/SwingView.java

```diff
@@ -28,6 +33,8 @@
     private ToolHandler _toolHandler;
     private SimulationController _simulationController;
     private SimPanel _simPanel;
+    private Simulation _simulation;
```

Comment:
The `Simulation` that is associated to the `SwingView` is currently stored in the `Settings` object `_settings`, so there's no need to also store it directly in the `SwingView`.

Further reading: The stored `Settings` object is immutable. This has the advantage that other code which accesses it don't have to care about synchronization.

Reply:
Thanks for the information!